src/coreComponents/dataRepository/Group.cpp

```diff
@@ -158,9 +161,14 @@ void Group::processInputFileRecursive( xmlWrapper::xmlNode & targetNode )
     else
     {
       // Make sure child names are not duplicated
-      GEOS_ERROR_IF( std::find( childNames.begin(), childNames.end(), childName ) != childNames.end(),
-                     GEOS_FMT( "Error: An XML block cannot contain children with duplicated names ({}/{}). ",
-                               getPath(), childName ) );
+      if( std::find( childNames.begin(), childNames.end(), childName ) != childNames.end() )
```

Comment:
why replacing GEOS_ERROR_IF ?

Reply:
Thanks for noticing it, it was for declaring a local variable but I forgot to change it back.